chapters/en/Unit 4 - Mulitmodal Models/introduction.mdx

```diff
@@ -0,0 +1,146 @@
+# Introduction
+
+Welcome to the chapter on Fusion of Text and Vision. This chapter builds the foundation
+for the later sections of the unit. We will explore:
+- The notion of multimodality, and different sensory inputs humans use for efficient decision making. 
+- Why is it important for making innovative applications and services through which we can interact and make lives easier.
+- Multimodality in context to Deep Learning, data, tasks, and models.
+- Some applications of multimodality, like multimodal emotion recognition and multimodal search.
+
+So let's begin 🤗
+
+## What is multimodality? 📸📝🎵
+
+As the word suggests, modality means a medium or way in which something exists or is done. 
+In our daily life, we come across many scenarios where we have to make decisions and perform tasks.
+For this we use our 5 sense organs (eyes to see, ears to hear, nose to smell, tongue to taste and skin to touch).
+Based on the information from all organs, we assess our environment, perform tasks and take decisions for our survival. Each of these 5 sense organs
+are 5 different modalities through which information comes to us and thus the word multimodality or multimodal. 
+
```

Comment:
Add a line about what's multi-modal in mathematics? like what's mode in math basically the multiple peak nodes? WDYT?

Reply:
Actually, it is more related to sensor fusion but if possible I will add a line 
Thanks @charchit7 :D